### PR TITLE
Removes Stable Mutagen

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Removes both chemistry recipes for Stable Mutagen, and makes it unable to be synthesised by botany. (Still admin spawnable)
Also makes any species with the `NOTRANSSTING` trait (Vox and Plasmamen) immune to the Changeling Transformation Sting in order to balance it out.

## Why It's Good For The Game

Stable Mutagen currently has two uses:

### 1. Vox Cloning:
For this use, not only does it pretty much invalidate the 'Cannot be cloned' malus, it also doesn't make any sense lore wise.
Cloning a Vox with SM also clones their Cortical Stack, which is a bio-mechanical implant. You can't copy robotic limbs and organs with SM, so there's no reason for this to be an exception.
And of course there's also this section on the Vox wiki page:
> So far, attempts at lab-grown samples have produced no results with such efforts often producing highly acidic compounds capable of burning through most lab equipment.

As it is that statement makes no sense, since anyone with medbay access and a bit of lithium can clone a perfectly functional Vox with no issues.

Removing Stable Mutagen *will* force gibbed Vox to play as another species for the remainder of the round, but considering that Slime People (The only other non-clonable species) have to do this already, I can't see this as too much of a downside.

### 2. Forcefully transforming half the crew into yourself:
I don't think I need to go into too much detail as to why this is bad, but suffice it to say that non-antags really shouldn't have access to a DIY Transformation Sting. Especially considering how controversial the sting itself already is.

## Changelog
:cl:
del: Removed the recipe for Stable Mutagen.
tweak: Made Vox and Plasmamen immune to the Changeling Transformation Sting.
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Im ok with this.

As for first point, yeah that doesnt make sense. Why should a cybernetic thingy be replicatable, and it invalidates the noclone part

As for the second, given we have dished out notes for people stable mutagen spamming people, and after seeing 30 of the same person in one shift (not to mention the hassle it causes for security), this has my design approval.

**Comment by ppi13** [comment]
Yes please, stable mutagen is a plague.
Slime people can be created with some frost oil and a funny looking slime from xenobio

**Review by dearmochi — APPROVED** [review]

**Comment by AzuleUtama** [comment]
Almost every time i've seen stable mutagen being used in-game it's for griefing. It's a neat idea for a chem but with all the problems it's brought I don't see a reason to keep it, at the most it should be admin spawnable only.

**Comment by Carthusia** [comment]
Im a fan of the idea, but im worried about the 5 minute heart timer due to how arbitrary its always felt to me, more so when it was 2.5 minutes.

5 minutes go by, and boop, your done, no more being fixed for you. Doesnt matter how much effort they put into your, if they fix every single bit of you, if they fix your heart or fully replace with with an advanced cybernetic replacement, your just still done as the heart you put in instantly decays after if you try to defib them after the 5 minute timer is up.

Id love to see this pair'd up with just the flat out removal of cloning, or at least the massive increase in its tech cost and resource cost per clone, make it somthing RND has to work towards and build instead of being a round start thing where you can produce enough biomass to last the entire round in 2 minutes if you know what to do with starter resources, and the removal of the 5 minute timer and the inclusion of proper surgery methods to fix people up enough so that they can be brought back up through difficult/complex surgical operations that become more resource intensive and time consuming the more decay'd the body is, with it becoming eventually impossible to save the body once it becomes a skeleton.

Also: Removing Stable Mutagen will force gibbed Vox to play as another species for the remainder of the round

Without SM, how would they play as another species if you cannot create new bodies for them to be transplanted into? This kind of has the side effect of almost removing the ability to do brain transplants into new hosts that are not borgs or monkeys, unless your using corpses from the morgue.

**Comment by procdrone** [comment]
Not a fan. SM has its uses in copying DNA of a person for brain transplantation revival. which is easier than spending whole round fixing one non-clone body whom all of organs have died and you have to literally mito every organ and limb, while dealing with patient constantly dying and accumulating genetics damage. (in the most severe cases).

As for the vox argument.:
A) Code makes lore, not vice versa on Paradise;
B) No clone malus, the vox were balanced around space proof, and the whole balance for that species is in my opinion skewed.
This revival method for vox is still a lot more involved than simply slapping into the cloner and waiting, so I wouldn't say its "cheating the system" that much. And the no decay "perk" doesnt affect SR effects so they still come out half septic. So I dont see this as being that much of a problem;
C) Slime people can turn back into slime people with help of Xenobiology;

Part 2 of the PR seems more of a Administrative issue that should be dealt with offenders rather than banned in code. self antagging people will simply find new ways to be self antagging.

**Comment by Mitchs98** [comment]
100% Agree with this PR. Lore reasoning aside, in the very vast majority of situations Stable Mutagen is used for griefing. I have seen few incidents of it actually being used for good, fewer of it being used for any sort of interesting gimmick. Most cases I see someone using it in a grenade, injecting it into people or food, or outright forcing it upon people. In any of these cases it's just not fun and straight up grief.

It's only 'good' use to clone Vox in my opinion is an argument of laziness. We SR'd and fixed up vox _just fine_ before it was added, we can do it again just fine once it's removed. This is essentially the same thing as botany players believing they NEED a portable chem machine to do their jobs effectively. They don't, and medbay doesn't need SM either.

**Comment by SabreML** [comment]
> Without SM, how would they play as another species if you cannot create new bodies for them to be transplanted into?

> SM has its uses in copying DNA of a person for brain transplantation revival. which is easier than spending whole round fixing one non-clone body whom all of organs have died and you have to literally mito every organ and limb, while dealing with patient constantly dying and accumulating genetics damage. (in the most severe cases).

This is by far the biggest issue with this PR, I think. All other playable species are either clonable or have some way to get another body of their species for a transplant. (Plasmamen are clonable, Slime people can get bodies from xenobio, etc.)

Ignoring lore stuff for a moment, removing this for vox is really just an unfair disadvantage to them. Because again, every other species in the game has some way to clone themselves. Usually if someone saves up 45 karma for a species, makes a character, etc, they want to *stay* as that species rather than having to be a Tajaran because they were one tile too close to an explosion.

Honestly I can't see a way to solve the gibbing issue at the moment without either a *massive* lore/balancing rewrite or some sort of 'Insert stack, get Vox' machine. Both of which would be impossible for their own reasons.

**Comment by Mitchs98** [comment]
> Slime people can get bodies from xenobio

I just want to point out this rarely, if ever, happens in practice. I've only ever witnessed it a handful of times in my playtime here.

**Comment by IIWOLFII** [comment]
just add dna record into same box with medical records and security records so if someone using it for griefing (also do we not have admins why is this even a point) CMO can just do boopity boop on dna computer and get it back in a syringe or something for person to turn back into self

adding dna records also will allow for antags to fuck with that so thats good

i don't see how removing ability to get new body of the same species you were is any fun and saying that this is also an issue with slimes makes no difference - its an _issue_ for a reason that needs to get fixed and not dogpiled on

**Comment by Mitchs98** [comment]
> just add dna record into same box with medical records and security records so if someone using stable mutagen for griefing (also do we not have admins why is this even a point) CMO can just do boopity boop on dna computer and get it back in a syringe or something for person to turn back into self
> 
> adding dna records also will allow for antags to fuck with that so thats good

I'm highly skeptical this would actually be a good idea. Not only would antags be able to get free syringes to dick around with as they please, it doesn't address the issue of it trivializing medbay. Nor should the DNA computer actually be able to synthesize some DNA. Gameplay > Lore, sure, you can argue that. But I happen to like what little uniqueness races have left, and little bits like this counts.

> stable mutagen for griefing (also do we not have admins why is this even a point)

It's been a valid point in the past for other pressing issues, and is a valid point now. If something can be done to lessen staff workload, and the feature itself is mostly used to troll or grief with, then why not remove it? There are very little downsides to removing Stable Mutagen, the most pressing one being vox bodies.

Vox players dealt with it in the past. Medbay dealt with properly SRing and treating Vox in the past. At this point it just seems like entitlement to want to keep it around because they got an easy route of revival. Which, they shouldn't have. They're unclonable for a reason. This is coming from someone that plays slime person, another unclonable race(bringing up the xenobio slime body thing has barely any relevance as it essentially never happens in practice), that has to either deal with a transplant or not play when they get gibbed.

If people are so keen on Vox getting a method of revival/cloning(still don't think they need it, it defeats the purpose and tosses the lore into the trash for no reason other than 'my easier gameplay'), then something else that's not grief bait could be added later down the line to compensate. 

Stable Mutagen has been a constant issue since it's addition with very little positive gain.

**Comment by procdrone** [comment]
> Vox players dealt with it in the past. Medbay dealt with properly SRing and treating Vox in the past. At this point it just seems like entitlement to want to keep it around because they got an easy route of revival. Which, they shouldn't have. They're unclonable for a reason. This is coming from someone that plays slime person, another unclonable race(bringing up the xenobio slime body thing has barely any relevance as it essentially never happens in practice), that has to either deal with a transplant or not play when they get gibbed.

Yeah, they dealt with it being morgue'd and not even revived. or gibbed by unskilful use of SR. Which was majority of cases. 

I played vox for a long time, and the only time i was properly revived (the only case i can remember) was an ADMIN taking pity on my agonizing plight (in dchat) at seeing medical not treating me properly even though my body was there for like 30 minutes.

Also it feels like you're missing an important point in all of this.

> At this point it just seems like entitlement to want to keep it around because they got an easy route of revival.

Of course, because its fairly fast (while still being more involved and complicated than cloning) compared to fixing a really fked up body by SR, and while maybe you are willing to spend half a round operating a single patient, other doctors might not so much, and definitely not everyone has the expertise to do it efficiently... or at all. not to mention other interruptions like attacks, other patients, and having an OR room occupied for half a round, or other random factors.

Also, the player on the receiving end is neither dead or alive through most of it. Having half your round critlocked/not even eligible for midrounds/ghost roles? People would like to play the game not suffer through prolonged and sloppy revivals. 

Most of the time you will just end up with a SSD body or a frustrated player, a waste of time.

And really, Vox have **no boons** to really justify these excruciating experiences for the players that play them.

**Comment by Mitchs98** [comment]
> Yeah, they dealt with it being morgue'd and not even revived. or gibbed by unskilful use of SR. Which was majority of cases.

> Of course, because its fairly fast (while still being more involved and complicated than cloning) compared to fixing a really fked up body by SR, and while maybe you are willing to spend half a round operating a single patient, other doctors might not so much, and definitely not everyone has the expertise to do it efficiently... or at all. not to mention other interruptions like attacks, other patients, and having an OR room occupied for half a round, or other random factors.

Slime players have to deal with the _exact_ same thing or get a brain transplant. What you're doing is ignoring this fact solely because Vox now have a fast and reliable method of revival that they should not of gained in the first place. Slime and Vox players _should have to deal with this._ One of their malus' are, in-fact, being harder to revive. Sure, it sucks. But that's just how it worked, and should still work.

> And really, Vox have **no boons** to really justify these excruciating experiences for the players that play them.

Perhaps players should play them for the lore and racial aspect, rather than the boons. I, personally, would prefer many of the slime changes reverted and them being simply nobreathe once again before you argue that 'You play slime they have lots of boons!'. I didn't ask for them, I don't care if they get removed.

**Comment by Carthusia** [comment]
> And really, Vox have **no boons** to really justify these excruciating experiences for the players that play them.

To be fair, i'd guess most people who play vox as a main or secondary understand its going to be more difficult and enjoy that fact. Same as those who play IPC and break by being brushed up against, or plasmamen who take 50% extra everything. Its part of what makes playing those races special, its more difficult which changes how you play them. You wouldnt play a human the same as a plasmaman when it comes to taking risks knowing how much easier your going to die should anything go wrong.

**Comment by Kiyahitayika** [comment]
@SabreML 
Regarding Point 1, the cortical stack that is "copied" isn't a copy. It's a `new` "brain" organ for the mob. That's like when you SM a Human or a Skrell. There's a brain because that's how the mob is initialized. To address this issue, you might be able to do something about removing `new_vox.has_organ["cortical stack"]` or something?

Regarding Point 2, do you think this could be addressed by checking to see if the mob-to-be-transformed has a client/mind attached? That way you could have the code return out if and only transform clientless/mindless mobs.


That said, removing SM? I'm all for it!

-----------
>  And really, Vox have **no boons** to really justify these excruciating experiences for the players that play them.

To be fair, Vox not skeletonizing leave them open to being revived with SR at any point during the round. This is a really awesome boon and closes that gap.

> Slime players have to deal with the exact same thing or get a brain transplant.

Yep! And that's fair, too, and the Slimepeople's boon of regrowing limbs makes SR revival pretty neat - "oh that limb is dead? let's amputate, feed you some stuff, hook up a slime jelly IV".

I think the worst part with SR is when Medbay has to do a bunch of invasive surgeries to repair dead limbs, but by the time you're needing to SR someone, Chemistry likely already has the meds you need.

I know @Marginalorb and others are still working on Wryn and there's a closed PR on the Paradise Github for Wryn Larvae. We merged it on Scorpio where I tested it on a live environment a few times and it worked, so doing a brain transplant or something like that would be a viable option for Wryn - when/if they become initialized on Paradise, hit me up and I'll resubmit that PR for y'all.

**Comment by SabreML** [comment]
I've released another PR (#15837) which will hopefully help with the issue of Strange Reagent revival being far too tedious compared to Stable Mutagen, so assuming that goes through it should solve that problem at least.

**Comment by IIWOLFII** [comment]
> I'm highly skeptical this would actually be a good idea. Not only would antags be able to get free syringes to dick around with as they please,

~~It could be those genetics syringes that u cant launch out of syringe gun, or you would order them to be arrived on cargo shuttle in singular amounts with point cost, or amount of crew with dna records grows as geneticists work etc etc, its just an idea for a workaround it could be made poorly or could be made well and would probably require separate PR and balancing etc~~
Nevermind, yeah thats bit too much work just for single chem. 

>>_stable mutagen for griefing (also do we not have admins why is this even a point)_

> It's been a valid point in the past for other pressing issues, and is a valid point now. If something can be done to lessen staff workload, and the feature itself is mostly used to troll or grief with, then why not remove it?

true, could be made less accessible by making it suck ass to make to discourage griefing

>There are very little downsides to removing Stable Mutagen, the most pressing one being vox bodies.

Its just that it can be improved instead of removed. Plus i thought this also works on slime people who got same noclone thing?

> Vox players dealt with it in the past. Medbay dealt with properly SRing and treating Vox in the past. At this point it just seems like entitlement to want to keep it around because they got an easy route of revival.

>Which, they shouldn't have.

Why
 
>They're unclonable for a reason.

I assume that the reason is to keep them alien or whatever, and i think that race uniqueness should be RPd not technical. Slime people can't break bones and take more brain damage but they are still nothing more but farting transparent humans for the most part. Vulps and tajs are just fluffy humans as well. If anything vox are more alien than others because funny language, inviolate, >xd smal raptor bird and _just partially_ because of brute damage.
>This is coming from someone that plays slime person, another unclonable race(bringing up the xenobio slime body thing has barely any relevance as it essentially never happens in practice), 

>Slime and Vox players should have to deal with this. One of their malus' are, in-fact, being harder to revive.
>that has to either deal with a transplant or not play when they get gibbed.
>not play when they get gibbed.

People already are salty when they die and this does not help
I cant say im a fan of playing as karma race only not to play as the karma race in case medbay bomb bonks me. Both vox and slimes could have interesting different weird complicated ways to replace cloning, its just that noone tries thinking of them because for some reason people think that noclone justifies inability to get spare body. If anything they both will feel more >alien and  >unique if slime people would grow their torsos back from their pinky finger while being marinated in jar with carpjuice mixed with cola and clown tears while placed in room with cool temperature, stir frequently, add salt and pepper to taste, serve hot
Harder to revive = good
Only being able to revive in human/tajar/monke body = stinky 

> If people are so keen on Vox getting a method of revival/cloning(still don't think they need it, it defeats the purpose and tosses the lore into the trash for no reason other than 'my easier gameplay'), then something else that's not grief bait could be added later down the line to compensate.

i thought this also works for slime peoples, oops. Well again maybe if it wouldn't be made in two clicks people wouldn't grief as much. 
Tbh people **will** find ways to grief wherever they can, its just that this way is very straightforward and not hard which can be fixed by making it not straightforward and hard to make, aka the recipe requiring some side element to be heated to 1000k and it evaportaing on creation if there is no stabilizing reagent, then cooling back down to 40 k in order to mix with something else and so on, and after hoops you get only 5 u on creation from full 100u beaker of ingridients
If someone spends 2 hours making these and only gets eight syringes then its bad enough not to grief with

> Stable Mutagen has been a constant issue since it's addition with very little positive gain.

I only saw it being used to dupe a dude on april first round and never ever again, i guess i missed out on rounds where someone used it to grief people, but once agian make it suck ass to make and noone will care enough to mass dupe same guy.

TL;DR I want it to stay because i like funny chems and i dont like how slimes and vox don't have any way to get spare bodies, even if it would be cancer to create them then its still better than having no option at all, to prevent griefing with SM just make it suck ass to produce and i think that griefing wouldn't be nearly as much of an issue. Plus i believe that this chem can be used to frame people by tators or in other creative ways and thats good to have

**Review by Kyep — APPROVED** [review]

**Comment by Mitchs98** [comment]
> true, could be made less accessible by making it suck ass to make to discourage griefing

In what manner would this even be possible without making the chem useless?

> Its just that it can be improved instead of removed.

This absolutely does not apply to everything, not even by a long shot, and really needs to stop being used as a buzzphrase. 'Improve don't remove' is slowly becoming little more than a meme or something people quote when they don't want a feature _they_ enjoy to be removed, rather than taking into account the net gain for it actually being removed or the effort it would be needed to 'improve' it.

> Plus i thought this also works on slime people who got same noclone thing?

Slime people do not have blood. No I would not agree with keeping it even if they did, nor do I want it to work on them.

> and i think that race uniqueness should be RPd not technical.

Technical quirks are needed to make the race more than human+. It absolutely shouldn't be RP only, by any means at all. The races DID used to be more varied and unique until newcrit hit, but that's a can of worms I'm not going to get into nor start discussing again as it's like beating a dead horse that's already a fine red paste into nothing more than mere atoms.

> People already are salty when they die and this does not help

Then they can man up and realize this is a part of the game. Sure I dislike dying too, but thems the brakes.

> Both vox and slimes could have interesting different weird complicated ways to replace cloning, its just that noone tries thinking of them because for some reason people think that noclone justifies inability to get spare body.

Actually neither race were meant to be able to be revived in the first place(this was before I played), Strange Reagent was then added to the game that allowed their revival(Ported from...somewhere, I forget where, before I played.). NOCLONE absolutely 100% is meant to prevent getting a spare body outside of being transplanted, that's quite literally what the variable was created for. I'm finally beginning to understand why the older players are saying us newer players should be lucky we have Strange Reagent now.


> some reason people think that noclone justifies inability to get spare body. If anything they both will feel more >alien and >unique if slime people would grow their torsos back from their pinky finger while being marinated in jar with carpjuice mixed with cola and clown tears while placed in room with cool temperature, stir frequently, add salt and pepper to taste, serve hot

This actually makes no sense whatsoever to the lore, and I would never want it to whatsoever either. Not to mention it's far to much of a bad meme, and more fitting for a server like Goon or TG.

> Harder to revive = good
> Only being able to revive in human/tajar/monke body = stinky

Again. Strange Reagent. Defibbing. These exist. And so what if you have to play a different race if you die? Sure it sucks, but that's what you signed up for when picking the race. No one made you pick the race, you picked it. You get what you get.

> Tbh people **will** find ways to grief wherever they can

This is an _incredibly_ poor reason to keep something around when its essentially only ever used to grief, in the semi-rare instances it's used to revive a Vox or a poor overused meme that no one really likes to deal with anymore.


> TL;DR I want it to stay because i like funny chems

Therein lies the problem, and one of the main reasons I feel you want SM to stay. It's not a funny chem, it genuinely isn't. It's a stale, very bad(was to begin with), meme that most people are tired of putting up with and groan everytime it's done. 

> Plus i believe that this chem can be used to frame people by tators or in other creative ways and thats good to have

An extremely rare, and far fetched, example that doesn't justify keeping a grief chem around.


> i dont like how slimes and vox don't have any way to get spare bodies,

This is the way it was for years, and the way it should of stayed. People are getting far to used to more hugboxed features being added in. Be it this, portable chemistry machines for botany, or anything in the future. This sentiment is why I'm vaguely in favor of cloning getting removed if I knew it would just permakill basically everyone due to medbay being far too lazy to do much more than clone people if they're more damaged than they want to bother fixing.

**Review by AffectedArc07 — APPROVED** [review]

**Comment by cluwneythescourge** [comment]
I guess this is late but I wanted to note that the PR is missing one of the actual good uses for stable mutagen.   If you've played Genetics, you know that many of the "big" powers like Telekinesis, Hulk, etc can be dormant, meaning that you might inject it and not see any effects.   Thus, a geneticist might think they have a blank that is actually a power.

The counter to this was to inject yourself with 10u SM and 5u of your own/the subjects blood.   This would trigger any dormant powers in the subject 100% of the time, giving a geneticist a way to test their blanks.   As genetics is already very tedious for not a whole lot of benefit (compared to the TG version) I would hope for a way around this if SM is being removed - maybe have a nerfed version of the chem, make it only available to Genetics, make it only available to CMO to hand out to genetics, or something - but please, removing it entirely is very sad for geneticists that know their job.

**Comment by Sikroth** [comment]
Can't upvote this enough even if it's already merged, god riddance. SM was both super grieffable and also dumb OP for medical uses.

As for the genetics thing, the "dormant" powers thing should just go away imho. Finding genetic mutations is already RNG enough without having to also suffer with wether a block is empty or just decided to not activate 8 times in a row. But, outside of the scope of this.

**Comment by cluwneythescourge** [comment]
WOuld be perfectly happy with getting rid of the "dormant' bullshit

**Comment by KozackWolf** [comment]
I worry about the requirement of SR. Omnizine itself is a pain to get without the hypospray. If the CMO is an antag, and hands that thing over without removing the omni, or hell, someone steals it early shift. You have to rely on donks or botany. Botany is one of the most unreliable departments to do what you need them to do, too busy making deathnettle and drugs. Donks may already be eaten. Maybe add a small map edit that gives chemistry a small bottle of omnizine since now it's the ONLY way to revive vox?

**Comment by KozackWolf** [comment]
> 
> 
> I guess this is late but I wanted to note that the comments on this PR seem to be missing one of the actual "legit" uses for stable mutagen. If you've played Genetics, you know that many of the "big" powers like Telekinesis, Hulk, Cloak of Shadows, Chameleon, No Breath, and more, can often be dormant, meaning that you might inject it and don't see any effects. Thus, a geneticist can't really tell if the block they are working is a blank or just a power that didn't activate.
> 
> The counter to this was to inject yourself with 10u SM and 5u of your own/the subjects blood. This would trigger any dormant powers in the subject 100% of the time, giving a geneticist a way to test their blanks. As genetics is already quite tedious for not a whole lot of benefit (especially compared to the TG/Goon versions, and at least for non-antags) I would hope for a way around this if SM is being removed - maybe have a nerfed version of the chem, make it only available to Genetics, make it only available to CMO to hand out to genetics, have powers always manifest visibly on lesser species, just to throw out some ideas off the top of the head - but there needs to be something here, as removing it entirely is very sad for geneticists that know their job.
> 
> One more note is that this also makes it impossible for a traitor/cult/vamp/etc geneticist to reliably hand out good powers to their allies, or even for a legit geneticist to do it when called for - it will be the case of 3 sec coming to get the power once approved, then only 1 of them actually manifesting it once injected, with no way to give it to the others and the Geneticist having to awkwardly explain how dormant powers work, assuming they even know.
> 
> My personal preferred solution as a player would be to leave SM uncraftabe but give genetics a supply in a freezer or something so they can be held accountable if it gets misused, for what that's worth.

This is why you usually have a second geneticist to work with. Test the blanks by tossing your partner in the machine.

**Comment by Mitchs98** [comment]
> I worry about the requirement of SR. Omnizine itself is a pain to get without the hypospray. If the CMO is an antag, and hands that thing over without removing the omni, or hell, someone steals it early shift. You have to rely on donks or botany. Botany is one of the most unreliable departments to do what you need them to do, too busy making deathnettle and drugs. Donks may already be eaten. Maybe add a small map edit that gives chemistry a small bottle of omnizine since now it's the ONLY way to revive vox?

This was always the case before SM was added in, and always was the case for slime players. I see no reason for it to change, nor do I see a reason for Vox to get special treatment(and I definitely don't want an easier method to revive slimes). There are also multiple donk pocket boxes on station and you can always cook more. People need to stop trying to trivialize a racial downside to the point it may as well not exist.

**Comment by KozackWolf** [comment]
Oh my god! I just realized the biggest travesty of removing SM... we can no longer make KFV without sentient vox!

**Comment by Mitchs98** [comment]
> Oh my god! I just realized the biggest travesty of removing SM... we can no longer make KFV without sentient vox!

I consider this a good thing.
